src/adapters/elementToTree.js

```diff
@@ -4,7 +4,7 @@ function nodeTypeFromType(type) {
   if (typeof type === 'string') {
     return 'host';
   }
-  if (type && type.prototype && typeof type.prototype.render === 'function') {
+  if (type && type.prototype && type.prototype.isReactComponent) {
```

Comment:
@ljharb i tried that, but it `isReactComponent` got introduced after react 13.  I'm thinking that when i refactor this code into separate code modules per adapter, i'll use render for react 13 and just `isReactComponent` after that.